docs/extend/extend-apiml/api-mediation-versioning.md

```diff
@@ -1,133 +1,104 @@
-# API Versioning
+# Understanding API Versioning
```

Comment:
I don't think we should touch this article unless we are willing to rewrite it from the scratch. There are too many incorrect information. 

If we want to keep this article, we need to write a short one that explains that it's possible and meaningful to version the APIs. 
Showcase how the API ML routes to the correct versions. 
Explain that for API ML only major version matters. 
Point to the right place in documentation with respect to metadata when onboarding service, that handles this area.

Reply:
The limitation of the content is good. I am ok with it.